src/simple.jl

```diff
@@ -1,16 +1,33 @@
 """
-    struct SimpleTimes <: TimeStructure
-A simple time structure conisisting of consecutive time periods of varying duration
+    struct SimpleTimes{T} <: TimeStructure{T}
+
+    SimpleTimes(len::Integer, duration::Vector{T}) where {T}
+    SimpleTimes(len::Integer, duration::Number)
+    SimpleTimes(dur::Vector{T}) where {T<:Number}
+
+A simple time structure conisisting of consecutive time periods of varying duration.
+`SimpleTimes` is always the lowest level in a `TimeStruct` time structure.
```

Comment:
It is also possible to have `CalendarTimes` at the lowest level. So maybe add a "(or CalendarTimes)".